### ISSUE TITLE
Error at untigging step

### ISSUE THREAD

**Issue by devenderarora** [marbl/canu#1723]
Hello Canu Team,
I am using Canu for genome assembly and using this command

```
canu -p minipig -d canu_assem genomeSize=2500m -pacbio-raw /680_info4/project/arora/canu/subreads_minipig.fasta useGrid=false ovsMethod=sequential gnuplotTested=true maxMemory=350g &> mini1.log
```

Report stop after 

```
[UNITIGGING/MERS]
--
--  22-mers                                                                                           Fraction
--    Occurrences   NumMers                                                                         Unique Total
--       1-     1 1424017685 *******************************************************************--> 0.3663 0.0151
--       2-     2 179540263 ************************                                               0.4125 0.0189
--       3-     4 114592433 ***************                                                        0.4312 0.0212
--       5-     7  68457862 *********                                                              0.4492 0.0245
--       8-    11  62155335 ********                                                               0.4637 0.0286
--      12-    16  92733565 ************                                                           0.4796 0.0356
--      17-    22 183129011 *************************                                              0.5055 0.0518
--      23-    29 350838540 ************************************************                       0.5570 0.0959
--      30-    37 507426065 ********************************************************************** 0.6523 0.2029
--      38-    46 477897393 *****************************************************************      0.7832 0.3892
--      47-    56 270663461 *************************************                                  0.9003 0.5952
--      57-    67  91526767 ************                                                           0.9635 0.7301
--      68-    79  23221907 ***                                                                    0.9842 0.7827
--      80-    92   9859204 *                                                                      0.9895 0.7989
--      93-   106   6150480                                                                        0.9920 0.8075
--     107-   121   4161805                                                                        0.9935 0.8138
--     122-   137   3148397                                                                        0.9945 0.8188
--     138-   154   2435203                                                                        0.9953 0.8230
--     155-   172   1961126                                                                        0.9959 0.8267
--     173-   191   1668189                                                                        0.9964 0.8301
--     192-   211   1391216                                                                        0.9969 0.8333
--     212-   232   1191134                                                                        0.9972 0.8363
--     233-   254   1013828                                                                        0.9975 0.8391
--     255-   277    854971                                                                        0.9978 0.8417
--     278-   301    749490                                                                        0.9980 0.8441
--     302-   326    635450                                                                        0.9982 0.8463
--     327-   352    546775                                                                        0.9984 0.8484
--     353-   379    463888                                                                        0.9985 0.8504
--     380-   407    404593                                                                        0.9986 0.8522
--     408-   436    354343                                                                        0.9987 0.8539
--     437-   466    305508                                                                        0.9988 0.8555
--     467-   497    270276                                                                        0.9989 0.8569
--     498-   529    241793                                                                        0.9990 0.8583
--     530-   562    219509                                                                        0.9990 0.8596
--     563-   596    202548                                                                        0.9991 0.8609
--     597-   631    185982                                                                        0.9991 0.8621
--     632-   667    174753                                                                        0.9992 0.8633
--     668-   704    165599                                                                        0.9992 0.8645
--     705-   742    153202                                                                        0.9993 0.8657
--     743-   781    138082                                                                        0.9993 0.8669
--     782-   821    130308                                                                        0.9993 0.8680
--
--   104401694 (max occurrences)
-- 92797877595 (total mers, non-unique)
--  2463828693 (distinct mers, non-unique)
--  1424017685 (unique mers)

[UNITIGGING/OVERLAPS]
--   category            reads     %          read length        feature size or coverage  analysis
--   ----------------  -------  -------  ----------------------  ------------------------  --------------------
--   middle-missing       3742    0.08    19579.94 +- 10793.69      1493.04 +- 1976.81    (bad trimming)
--   middle-hump           172    0.00     5170.69 +- 4611.88        754.03 +- 1227.98    (bad trimming)
--   no-5-prime           1033    0.02     9037.74 +- 7813.18       1060.33 +- 1841.19    (bad trimming)
--   no-3-prime            963    0.02     8482.61 +- 7922.71       1109.16 +- 1800.50    (bad trimming)
--   
--   low-coverage         4867    0.10     6213.98 +- 5695.28          3.88 +- 1.98       (easy to assemble, potential for lower quality consensus)
--   unique            2500313   53.92    19975.62 +- 5595.06         32.27 +- 8.10       (easy to assemble, perfect, yay)
--   repeat-cont        212536    4.58    18388.41 +- 5922.49        438.56 +- 697.87     (potential for consensus errors, no impact on assembly)
--   repeat-dove          3008    0.06    34866.42 +- 8702.64        288.03 +- 459.40     (hard to assemble, likely won't assemble correctly or even at all)
--   
--   span-repeat        822564   17.74    21659.75 +- 6688.67       7322.21 +- 6656.03    (read spans a large repeat, usually easy to assemble)
--   uniq-repeat-cont   878029   18.94    19529.05 +- 4543.42                             (should be uniquely placed, low potential for consensus errors, no impact on assembly)
--   uniq-repeat-dove    90503    1.95    30617.36 +- 8529.47                             (will end contigs, potential to misassemble)
--   uniq-anchor        118502    2.56    21122.43 +- 6492.16       8494.68 +- 7277.09    (repeat read, with unique section, probable bad read)

[UNITIGGING/ADJUSTMENT]
-- No report available.

**Log file suggest**

-- Found PacBio corrected reads in 'unitigging/minipig.gkpStore'.
--
-- Generating assembly 'minipig' in '/680_info4/project/WCPark/canu/canu_assem'
--
-- Parameters:
--
--  genomeSize        2500000000
--
--  Overlap Generation Limits:
--    corOvlErrorRate 0.2400 ( 24.00%)
--    obtOvlErrorRate 0.0450 (  4.50%)
--    utgOvlErrorRate 0.0450 (  4.50%)
--
--  Overlap Processing Limits:
--    corErrorRate    0.3000 ( 30.00%)
--    obtErrorRate    0.0450 (  4.50%)
--    utgErrorRate    0.0450 (  4.50%)
--    cnsErrorRate    0.0750 (  7.50%)
--
--
-- BEGIN ASSEMBLY
--
-- Finished stage 'unitig', reset canuIteration.
--
-- Running jobs.  First attempt out of 2.
----------------------------------------
-- Starting 'bat' concurrent execution on Mon May 25 09:11:21 2020 with 11112.943 GB free disk space (1 processes; 4 concurrently)

    cd unitigging/4-unitigger
    ./unitigger.sh 1 > ./unitigger.000001.out 2>&1

-- Finished on Mon May 25 09:13:07 2020 (106 seconds) with 11112.142 GB free disk space
----------------------------------------
--
-- Bogart failed, retry
--
--
-- Running jobs.  Second attempt out of 2.
----------------------------------------
-- Starting 'bat' concurrent execution on Mon May 25 09:13:07 2020 with 11112.142 GB free disk space (1 processes; 4 concurrently)

    cd unitigging/4-unitigger
    ./unitigger.sh 1 > ./unitigger.000001.out 2>&1

-- Finished on Mon May 25 09:14:59 2020 (112 seconds) with 11111.719 GB free disk space
----------------------------------------
--
-- Bogart failed, tried 2 times, giving up.
--

ABORT:
ABORT: Canu 1.6
ABORT: Don't panic, but a mostly harmless error occurred and Canu stopped.
ABORT: Try restarting.  If that doesn't work, ask for help.
ABORT:
```

**Comment by skoren** [comment]
Version 1.6 is two years old so I'd rather not try to debug why it crashed, it may well be a bug that has since been fixed. I suggest re-running your assembly with the 2.0 release starting from the assembly step:
```
canu -assemble -p minipig -d canu_assem_v2.0 genomeSize=2500m -pacbio-corrected= canu_assem/minpig.trimmedReads.fasta.gz useGrid=false maxMemory=350g &> mini2.log
```

**Comment by devenderarora** [comment]
Dear Sir,
I am trying to run the mention command using canu 2.0. I am getting an error saying Parameter corrected is not known. Am I doing anything wrong with it -pacbio-**corrected**

**Comment by skoren** [comment]
There was an extra = in my previous command, otherwise it should work. If it's not post your full command and all the output from it.

**Comment by devenderarora** [comment]
It works. Thankyou so much sir for the quick response. Hope this will end up successfully. 
I am grateful for such nice and quick help.

**Comment by skoren** [comment]
Any update?

**Comment by devenderarora** [comment]
Yes Sir. The assembly part was soon completed. When we checked the stats we observed low coverage and total counts was not near to what we expected. We again re-run the assembly in canu2.0 and still the no. of counts and contigs were different but with high coverage. Currently we re-run the assembly process with little tweaks. We are waiting to finish the assembly.
Any Idea why the results being vary. We already ran Canu on related species with no such issue.

**Comment by skoren** [comment]
I'm not sure what you mean by low coverage, the coverage between 1.6 and 2.0 wouldn't change if they're using the same corrected/trimmed data. Your previous plots show reasonable coverage (30-37x coverage peak) and reasonable overlap statistics. Without more details on what exactly you mean by low coverage or counts and what tweaks you're using I can't provide any comment.

**Comment by skoren** [comment]
Idle, original issue resolved by switching to version 2.0 from 1.6. If you want to provide more information on the assemblies to get advice on parameters, open a new issue or see the FAQ.
